eng/templates/jobs/ci-unit-tests.yml

```diff
@@ -35,19 +35,22 @@ jobs:
         displayName: 'Install dependencies'
         condition: and(eq(variables.isSdkRelease, false), eq(variables.isExtensionsRelease, false), eq(variables['USETESTPYTHONSDK'], false), eq(variables['USETESTPYTHONEXTENSIONS'], false))
       - bash: |
-          echo "Python version from variable: $PYTHON_VERSION"
-          if (( $(echo "$PYTHON_VERSION < 3.13" | bc -l) )); then
-              echo "Running unittests (Python < 3.13)..."
+          PY_VER="$(PYTHON_VERSION)"
+          echo "Python version: $PY_VER"
+      
+          # Extract minor version as integers
+          PY_MINOR="${PY_VER#*.}"
+      
+          if [ "$PY_MINOR" -ge 13 ]  then
+              echo "Running proxy_worker tests (Python >= 3.13)..."
```

Comment:
NIT: Add the message with specific Python version for which you are running the tests.

echo "Running proxy_worker tests for Python $(PYTHON_VERSION).$(PY_MINOR)"

Reply:
This pipeline will be run for all versions greater than 3.13. Adding the major version in the message doesn't help much since the major version won't change